src/element/textWysiwyg.test.tsx

```diff
@@ -1504,5 +1505,36 @@ describe("textWysiwyg", () => {
         }),
       );
     });
+
+    it("should text be centered when binding to a rectangle", async () => {
+      const rectangle1 = API.createElement({
+        type: "rectangle",
+        id: "rectangle1",
+        width: 100,
+        height: 100,
+      });
+
+      const text1 = API.createElement({
+        type: "text",
+        id: "text1",
+        text: "Snorf",
+      });
+
+      h.elements = [rectangle1, text1];
+
+      API.setSelectedElements([rectangle1, text1]);
+
+      expect(h.state.selectedElementIds[(rectangle1.id, text1.id)]).toBe(true);
+
+      h.app.actionManager.executeAction(actionBindText);
```

Comment:
Actually we don't need another test for this, we already have one test which checks the whole flow by opening context menu and triggering the action so lets merge it there - https://github.com/excalidraw/excalidraw/blob/master/src/element/textWysiwyg.test.tsx#L743
just add the alignment and coords assertions in the same test

Reply:
Done.